### PR TITLE
Add mission, vision, strategy, and fix name typo

### PR DESCRIPTION
**Pull request by zanetworker**
This PR adds mission, vision, and strategy. It also fixes a typo in my name in the charter doc. 

closes https://github.com/cncf/tag-runtime/issues/129

### CODE REVIEW

**Review by raravena80 — COMMENTED** [review]
Can you move the files under `website/content/wgs/cnaigs` to under `wgs/cnaiwg` (root dir) then create just dir symlinks under `website/content/wgs/cnaigs` to the content under `wgs/cnaiwg` (root dir).   

Example under `website/content/wgs/cnaigs`  run `ln -s  ../../../../wgs/cnaiwg/mission_vission_strategy`.   You can wait until I merge https://github.com/cncf/tag-runtime/pull/136 which fixes some of the symlinks.

Let me know if you need any help with this. Thanks!

**Comment by zanetworker** [comment]
closes https://github.com/cncf/tag-runtime/issues/129

**Review by raravena80 — APPROVED** [review]
lgtm. Thank you!

**Pull request merged**
